### ISSUE TITLE
torrent creation for folder instead of mkv file

### ISSUE THREAD

**Issue by Badshah90** [L4GSP1KE/Upload-Assistant#308]
Hi,
is it possible for script to create torrent for entire folder instead of just the mkv file inside the folder.
having this option would be really great.
Thanks.

**Comment by sirius-sama** [comment]
This script can upload pack/folder.

**Comment by MasalaTalk** [comment]
I have each release with nfo, poster and screens. This script is making torrent only for mkv

**Comment by swannie-eire** [comment]
> I have each release with nfo, poster and screens. This script is making torrent only for mkv

Nfos, screens and posters are not to be included in the torrent files in most trackers.

**Comment by MasalaTalk** [comment]
> > I have each release with nfo, poster and screens. This script is making torrent only for mkv
> 
> Nfos, screens and posters are not to be included in the torrent files in most trackers.

yeah but I was saying having an option to include them is better where it's allowed. this way release remains intact.
